### ISSUE TITLE
Model scale not accurate

### ISSUE THREAD

**Issue by ApolloBian** [rixtox/Topre-to-Cherry-MX-Adapter#3]
I tried your solution and overall I'm very satisfied with the end result.
However I do notice that the keystem is too tight for many keycaps. 
I inspected the file and get the noticed the specification as below:
![e828e62ca5434b0f76743104995ee01](https://user-images.githubusercontent.com/13659369/69396151-4a98e180-0d1c-11ea-8887-8d5319250b09.png)
which does not conform to the standard cherry mx keystem size on [cherry mx dev page](https://www.cherrymx.de/en/dev.html):

![image](https://user-images.githubusercontent.com/13659369/69396196-74ea9f00-0d1c-11ea-9452-19fb2dbd67a2.png)

**Comment by rixtox** [comment]
I didn't know such public schematic exists!
I measured those values manually so they may differ from the standard.
I will update the models so they use the official values.

Thanks for the info!

**Comment by rixtox** [comment]
Do you think something like this would work better?
![](https://i.imgur.com/a1EoTnT.png)

**Comment by ApolloBian** [comment]
Hi @rixtox , thanks for the timely reply! IMO 1.15mm could be a better way to go, given you can print with 0.05mm accuracy printers. Maybe include 2 versions (1.15 and 1.10) is a good idea.

**Comment by ApolloBian** [comment]
May I ask you to update the modified models in your free time?

**Comment by rixtox** [comment]
> May I ask you to update the modified models in your free time?

I'm not sure making two versions of every adapters just for a 0.05mm clarance is a good idea. Especially it would cause a lot of confusion to people.

I think we should actually print them out and conduct an evaluation before choosing one.

**Comment by ApolloBian** [comment]
Good point. I'm actually planning to print the stems for a new keyboard. If you can send me the modified versions I'll be happy to help test it and give more feedback. (I'm not capable of modifying the models though)

**Comment by ApolloBian** [comment]
Hi @rixtox . I tried the 1.1mm version and it is very good. Also I'd like to point out that the diameter for the cylinder should be 8.2mm, and width of the feet should be 2.3mm and I recommend to update it, because the original model is too tight and hard to press and I have to sand it manually
![image](https://user-images.githubusercontent.com/13659369/70385777-5d254300-19cb-11ea-90a7-1527eafa7894.png)
![image](https://user-images.githubusercontent.com/13659369/70385784-6c0bf580-19cb-11ea-86ad-1581afedcfbf.png)

**Comment by driph** [comment]
Heya, has the repo been updated with the files with the accurate stems?
